fastscanner/tests/test_partitioned_csv_provider.py

```diff
@@ -0,0 +1,80 @@
+import os
+import json
+from datetime import date, datetime, time, timedelta 
+import pandas as pd
+import pytest
+from unittest.mock import MagicMock, patch
+from fastscanner.pkg.localize import LOCAL_TIMEZONE_STR
+import pytz
+
+tz = pytz.timezone(LOCAL_TIMEZONE_STR)
+from fastscanner.adapters.candle.partitioned_csv import PartitionedCSVBarsProvider
+from fastscanner.services.indicators.ports import CandleCol
+
+SYMBOL = "AAPL"
+FREQ = "1min"
+UNIT = "min"
+TEST_KEY = datetime.now().date().isoformat()
+SAMPLE_DF = pd.DataFrame({
+    CandleCol.DATETIME: pd.date_range("2023-01-01 00:00", periods=5, freq="2min", tz=tz)
+}).set_index(CandleCol.DATETIME)
+
+
+SAMPLE_DF[CandleCol.OPEN] = [1, 2, 3, 4, 5]
+SAMPLE_DF[CandleCol.HIGH] = [1, 2, 3, 4, 5]
+SAMPLE_DF[CandleCol.LOW] = [1, 2, 3, 4, 5]
+SAMPLE_DF[CandleCol.CLOSE] = [1, 2, 3, 4, 5]
+SAMPLE_DF[CandleCol.VOLUME] = [100, 200, 300, 400, 500]
```

Comment:
Let's use a different dataframe for each test. Different periods, different freq, etc.